aria-practices.html

```diff
@@ -6737,6 +6734,139 @@ <h3>Range properties with spin buttons</h3>
       <pre><code>&lt;label&gt;Price per paperclip: $&lt;input type="number" min="0.01" value="0.5" max="2" step="0.01"&gt;&lt;/label&gt;</code></pre>
     </section>
   </section>
+  
+  <section id="aria-level">
+    <h2>Describing Hierarchical Structure with <code>aria-level</code></h2>
+    <p>
+      When elements have a hierarchical relationship, such as headings of nested sections or data in tree structures, the value of <a href="#aria-level" class="property-reference">aria-level</a> communicates an element's level within the hierarchy to assistive technologies.
+      The value of <code>aria-level</code> is numeric, with <code>1</code> indicating the top level of the structure.
```

Comment:
Let's make it very clear that level is an integer, and not just "numeric" (because someone might think that they can use "1.2" or similar).

```suggestion
      The value of <code>aria-level</code> is an integer greater than or equal to 1, with <code>1</code> indicating the top level of the structure.
```